eventstore/models.py

```diff
@@ -107,6 +107,14 @@ class MSISDNSwitch(models.Model):
     data = JSONField(default=dict, blank=True, null=True)
 
 
+class DeliveryFailures(models.Model):
```

Comment:
It is a Django best practice to name the models in singular form, ie. `DeliveryFailure`.